pages/index.js

```diff
@@ -21,23 +22,27 @@ const goods = [
 
 function Home({ good }) {
```

Comment:
you don't need to destructure good at this point, since it does not come from another component, but is at the top level of the same file. It should work the same if you leave it out.
```suggestion
function Home() {
```